storage/gcs/src/integration-test/java/io/aiven/kafka/tieredstorage/storage/gcs/GcsStorageMetricsTest.java

```diff
@@ -108,7 +108,7 @@ void metricsShouldBeReported() throws StorageBackendException, IOException, JMEx
         storage.delete(key);
 
         final ObjectName gcsMetricsObjectName =
-            ObjectName.getInstance("aiven.kafka.server.tieredstorage.gcs:type=gcs-metrics");
+            ObjectName.getInstance("aiven.kafka.server.tieredstorage.gcs:type=gcs-storage-metrics");
```

Comment:
Weeeell, considering `gcs` is already "storage"... :)

Reply:
s3 + storage = s4 😂

Reply:
Going for s3-client-metrics instead. Let me know how it looks now